pkg/portal/cluster/clusteroperators_test.go

```diff
@@ -55,33 +56,173 @@ func TestClusterOperators(t *testing.T) {
+					{
+						Message: "OAuthServerDeploymentAvailable: availableReplicas==2",
+						Reason:  "AsExpected",
+						Status:  "True",
+						Type:    "Available",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Upgradeable",
+					},
+				},
 			},
 			{
 				Name:      "cluster-autoscaler",
 				Available: "True",
+				Conditions: []OperatorCondition{
+					{
+						Message: "at version 4.6.26",
+						Reason:  "AsExpected",
+						Status:  "True",
+						Type:    "Available",
+					},
+					{
+						Status: "False",
+						Type:   "Progressing",
+					},
+					{
+						Status: "False",
+						Type:   "Degraded",
+					},
+					{
+						Status: "True",
+						Type:   "Upgradeable",
+					},
+				},
 			},
 			{
 				Name:      "cloud-credential",
 				Available: "True",
+				Conditions: []OperatorCondition{
+					{
+						Status: "True",
+						Type:   "Available",
+					},
+					{
+						Status: "False",
+						Type:   "Degraded",
+					},
+					{
+						Status: "False",
+						Type:   "Progressing",
+					},
+					{
+						Status: "True",
+						Type:   "Upgradeable",
+					},
+				},
 			},
 			{
 				Name:      "config-operator",
 				Available: "True",
+				Conditions: []OperatorCondition{
+					{
+						Reason: "AsExpected",
+						Status: "False",
+						Type:   "Degraded",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "False",
+						Type:   "Progressing",
+					},
+					{
+
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Available",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Upgradeable",
+					},
+				},
 			},
 			{
 				Name:      "console",
 				Available: "True",
+				Conditions: []OperatorCondition{
+					{
+						Reason: "AsExpected",
+						Status: "False",
+						Type:   "Degraded",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "False",
+						Type:   "Progressing",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Available",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Upgradeable",
+					},
+				},
 			},
 			{
 				Name:      "aro",
 				Available: "True",
+				Conditions: []OperatorCondition{
+					{
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Available",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "False",
+						Type:   "Progressing",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "False",
+						Type:   "Degraded",
+					},
+					{
+						Reason: "AsExpected",
+						Status: "True",
+						Type:   "Upgradeable",
+					},
+				},
 			},
 		},
 	}
 
 	sort.SliceStable(info.Operators, func(i, j int) bool { return info.Operators[i].Name < info.Operators[j].Name })
 	sort.SliceStable(expected.Operators, func(i, j int) bool { return expected.Operators[i].Name < expected.Operators[j].Name })
 
+	layout := "2006-01-02 15:04:05 -0700 MST"
+	for i, r := range info.Operators {
```

Comment:
You could possibly make use of parallel tests here: https://clavinjune.dev/en/blogs/speed-up-golang-test-execution-time/

Reply:
This is a singular test, what do you mean by this here?

Reply:
And a small warning too: https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721

Reply:
This is undeniabally an optimisation on time, but it feels like an unneccessary change given that the entire portal suite takes 100ms to run and the entire ARO-RP unit test suite takes 60 seconds.

I'd like to hear opinions from others in regards if we want to start thinking about time optimisation for unit testing?

Reply:
I don't think it's that big a change to warrant caring one way or the other... but here's a code snippet for it to encourage having the change:
```
	for i, r := range info.Operators {
		// Ensure vars are accessible to goroutine in t.Run
		r := r
		i := i
		t.Run(r.Name, func(t *testing.T) {
			t.Parallel()
			for j, c := range r.Conditions {
				// Ensure vars are accessible to goroutine in t.Run
				c := c
				j := j
				t.Run(c.Type, func(t *testing.T) {
					t.Parallel()
					_, err := time.Parse(layout, c.LastUpdated)
					if err != nil {
						t.Error("Given date was not in the correct format " + layout + err.Error())
						continue
					}
					info.Operators[i].Conditions[j].LastUpdated = ""
				})
				
			}
		})
	}
```

Reply:
Done!